blog/en/blog/2023/01/18/consul-with-apisix.md

```diff
@@ -0,0 +1,274 @@
+---
+title: "How to Integrate API Gateway and Consul? Not Consul K/V"
+authors:
+  - name: "Yihao LI"
+    title: "Author"
+    url: "https://github.com/Fabriceli"
+    image_url: "https://github.com/Fabriceli.png"
+keywords:
+- API Gateway
+- Consul
+- Service Discovery
+- Service Register
+description: Apache APISIX supports the Consul service discovery registry. This article will walk you through the process of implementing service discovery and service registry in APISIX.
+tags: [Ecosystem]
+image: https://static.apiseven.com/2022/blog/0818/ecosystem/HashiCorp%20Consul.png
+---
+
```

Comment:
1. Need a `<!--truncate-->` here to cut actual contents off from the blog summary page
2. Need a sentence or two to add a summary for this blog.
```suggestion
> This article teaches you how to integrate Consul with Apache APISIX 
<!--truncate-->

```
Now it looks like this, and it is just not right.

![image](https://user-images.githubusercontent.com/36651058/213606541-05dafa7c-9119-4419-99be-df1a293d1f63.png)

Reply:
done